src/client/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.Core.v3;
+
+namespace NuGet.CommandLine.StagingExtensions
+{
+    [Command(resourceType: typeof(StagingResources), commandName: "stage", descriptionResourceName: "StageCommandDescription",
+       MinArgs = 0, MaxArgs = 0, UsageSummaryResourceName = "StageCommandUsageSummary")]
+    public class StageCommand : Command
+    {
+        [Option(typeof(StagingResources), "StageCommandCreateDescription")]
+        public string Create { get; set; }
+
+        [Option(typeof(StagingResources), "StageCommandDropDescription")]
+        public string Drop { get; set; }

```

Comment:
We should have a default confirmation for drop (to avoid dataloss) and have a noninteractive support